### PR TITLE
[WIP] AAP-18144: Editorial edits

### PR DESCRIPTION
**Pull request by lmalivert**
Jira link: [AAP-18144](https://issues.redhat.com/browse/AAP-18144)

[AAP Install Guide](https://access.redhat.com/documentation/en-us/red_hat_ansible_automation_platform/2.4/html/red_hat_ansible_automation_platform_installation_guide/platform-system-requirements#doc-wrapper)

Incorporated editorial suggestions from [Chapter 2 System Requirements](https://docs.google.com/document/d/1tRNUZxvBDFxPbiPDVGWEywvwV2Eta_5mhk-CvbQIObc/edit#heading=h.hwwjfp2bkfct)

Files modified: 

- assemblies/platform/assembly-system-requirements.adoc
- modules/platform/proc-benchmark-postgresql.adoc
- modules/platform/proc-enable-hstore-extension.adoc
- modules/platform/ref-automation-hub-requirements.adoc
- modules/platform/ref-controller-system-requirements.adoc
- modules/platform/ref-ha-hub-reqs.adoc
- modules/platform/ref-postgresql-requirements.adoc
- modules/platform/ref-system-requirements.adoc

### CODE REVIEW

File: downstream/assemblies/platform/assembly-system-requirements.adoc [hunk 1]
```diff
@@ -9,9 +9,9 @@ Use this information when planning your {PlatformName} installations and designi
...
-* You must configure an NTP client on all nodes. For more information, see link:https://access.redhat.com/documentation/en-us/red_hat_ansible_automation_platform/{PlatformVers}/html/red_hat_ansible_automation_platform_upgrade_and_migration_guide/migrate-isolated-execution-nodes#automation_controller_configuration_requirements[Configuring NTP server using Chrony].
+* You can obtain root access either through the `sudo` command, or through privilege escalation. For more on privilege escalation see link:https://docs.ansible.com/ansible/latest/playbook_guide/playbooks_privilege_escalation.html[Understanding Privilege Escalation].
+* You can de-escalate privileges from root to users such as: AWX, PostgreSQL, {EDAName}, or Pulp.
+* You have configure an NTP client on all nodes. For more information, see link:https://access.redhat.com/documentation/en-us/red_hat_ansible_automation_platform/{PlatformVers}/html/red_hat_ansible_automation_platform_upgrade_and_migration_guide/migrate-isolated-execution-nodes#automation_controller_configuration_requirements[Configuring NTP server using Chrony].
```
Comment (jself-sudoku) [inline downstream/assemblies/platform/assembly-system-requirements.adoc:14]: Minor nit: 

```suggestion
* You have configured an NTP client on all nodes. For more information, see link:https://access.redhat.com/documentation/en-us/red_hat_ansible_automation_platform/{PlatformVers}/html/red_hat_ansible_automation_platform_upgrade_and_migration_guide/migrate-isolated-execution-nodes#automation_controller_configuration_requirements[Configuring NTP server using Chrony].d
```

File: downstream/modules/platform/proc-enable-hstore-extension.adoc [hunk 2]
```diff
@@ -6,7 +6,7 @@ From {PlatformNameShort} {PlatformVers}, the database migration script uses `hst
 
 This process is automatic when using the {PlatformNameShort} installer and a managed PostgreSQL server.
 
-However, when the PostgreSQL database is external, you must carry out this step manually before {HubName} installation.
+If the PostgreSQL database is external, you must enable the hstore extension to the {HubName} PostreSQL database manually before {HubName} installation.
```
Comment (jself-sudoku) [inline downstream/modules/platform/proc-enable-hstore-extension.adoc:9]: Minor nit. Should we add the ` before/afer hstore?

```suggestion
If the PostgreSQL database is external, you must enable the `hstore` extension to the {HubName} PostreSQL database manually before {HubName} installation.
```

File: downstream/modules/platform/ref-postgresql-requirements.adoc [hunk 3]
```diff
@@ -29,15 +28,12 @@ h| Service |Required |Notes
 * All {ControllerName} data is stored in the database.
 Database storage increases with the number of hosts managed, number of jobs run, number of facts stored in the fact cache, and number of tasks in any individual job.
 For example, a playbook run every hour (24 times a day) across 250 hosts, with 20 tasks, will store over 800000 events in the database every week.
-* If not enough space is reserved in the database, old job runs and facts must be cleaned on a regular basis.
-Refer to link:https://docs.ansible.com/ansible-tower/3.8.3/html/administration/management_jobs.html#ag-management-jobs[Management Jobs] in the _Automation Controller Administration Guide_ for more information
-
+* If not enough space is reserved in the database, old job runs and facts must be cleaned on a regular basis. Refer to link:https://docs.ansible.com/ansible-tower/3.8.3/html/administration/management_jobs.html#ag-management-jobs[Management Jobs] in the _Automation Controller Administration Guide_ for more information.
```
Comment (jself-sudoku) [inline downstream/modules/platform/ref-postgresql-requirements.adoc:31]: Very minor; seems like an article is needed. You can ignore this suggestion if you feel it's unnecessary.

```suggestion
* If not enough space is reserved in the database, the old job runs and facts must be cleaned on a regular basis. Refer to link:https://docs.ansible.com/ansible-tower/3.8.3/html/administration/management_jobs.html#ag-management-jobs[Management Jobs] in the _Automation Controller Administration Guide_ for more information.
```

File: downstream/modules/platform/ref-system-requirements.adoc [hunk 4]
```diff
@@ -12,13 +12,13 @@ Your system must meet the following minimum system requirements to install and r
 |===
 | Requirement | Required | Notes
 
-h| Subscription | Valid Red Hat Ansible Automation Platform |
+h| Subscription | Valid {PlatformName} |
 
 h| OS | Red Hat Enterprise Linux 8.6 or later 64-bit (x86, ppc64le, s390x, aarch64) |{PlatformName} is also supported on OpenShift, see link:https://access.redhat.com/documentation/en-us/red_hat_ansible_automation_platform/{PlatformVers}/html/deploying_the_red_hat_ansible_automation_platform_operator_on_openshift_container_platform/index[Deploying the Red Hat Ansible Automation Platform operator on OpenShift Container Platform] for more information.
```
Comment (jself-sudoku) [inline downstream/modules/platform/ref-system-requirements.adoc:17]: I think we're using the attribute for RHEL now.

```suggestion
h| OS | {RHEL} 8.6 or later 64-bit (x86, ppc64le, s390x, aarch64) |{PlatformName} is also supported on OpenShift, see link:https://access.redhat.com/documentation/en-us/red_hat_ansible_automation_platform/{PlatformVers}/html/deploying_the_red_hat_ansible_automation_platform_operator_on_openshift_container_platform/index[Deploying the Red Hat Ansible Automation Platform operator on OpenShift Container Platform] for more information.
```

Comment (lmalivert) [reply downstream/modules/platform/ref-system-requirements.adoc:17]: Thanks. That slipped my mind.

**Comment by jself-sudoku** [comment]
LGTM!

**Pull request merged**